PostgresV11.md

```diff
@@ -0,0 +1,98 @@
+### V11 postgres migration steps
+
+##  Pull latest ccd docker
+* Synch all micro-services with the same branch version for instance: develop
+* Start ccd docker and make sure that the new container ccd-shared-database-v11 is up and running.
+* uncomment ccd-shared-database-v11 section in the backend.yml
+````
+#This section should be commented when the pr is merged in master.
+  ccd-shared-database-v11:
```

Comment:
? 
don't we have to comment the old db?

Reply:
Yes , I could comment the old DB for this PR  and then we merge it  I comment the new DB, uncomment the old DB and comment the CCD_POSTGRES_11 VAR.  Should I do that ?

Reply:
The comments "#This section should be commented when the pr is merged in master." is a reminder for me that we should not merge in develop ccd-shared-database-v11: section .  

I removed this comments from there .  I am going to add these notes in another place